$NetBSD: patch-ad,v 1.4 2016/07/16 20:00:14 wiz Exp $

Allow passing in POSIX_SHELL from environment to avoid paths to tools/ dir.

--- configure.orig	2011-05-06 14:06:58.000000000 +0000
+++ configure
@@ -13518,6 +13518,7 @@ $as_echo "$ac_cv_path_SED" >&6; }
 
   while :
   do
+      test -x "$POSIX_SHELL" && break
       POSIX_SHELL=`which bash`
       test -x "$POSIX_SHELL" && break
       POSIX_SHELL=`which dash`
